charts/incubator/notifiarr/questions.yaml

```diff
@@ -0,0 +1,138 @@
+                                  group: "App Configuration"
+                                  label: "Image Environment"
+                                  schema:
+                                    additional_attrs: true
+                                    type: dict
+                                    attrs:
+                                      - variable: DN_API_KEY
+                                        label: "Notifiarr All API Key"
+                                        description: "Notifiarr All Integrations API key from https://notifiarr.com"
+                                        schema:
+                                          type: string
+                                          default: "CHANGEME"
+                                      - variable: upstream1
+                                        label: "Upstream 1"
+                                        description: "Controls which IP is written to the web server log file (requestor or x-forwarded-for header). Also allows the x-webauth-user header for proxy authentication. Separate with spaces. The default kubernetes network is pre-filled."
+                                        schema:
+                                          type: string
+                                          default: ""
+                                      - variable: upstream2
+                                        label: "Upstream 2"
+                                        description: "Controls which IP is written to the web server log file (requestor or x-forwarded-for header). Also allows the x-webauth-user header for proxy authentication. Separate with spaces. The default kubernetes network is pre-filled."
+                                        schema:
+                                          type: string
+                                          default: ""
+                                      - variable: upstream3
+                                        label: "Upstream 3"
+                                        description: "Controls which IP is written to the web server log file (requestor or x-forwarded-for header). Also allows the x-webauth-user header for proxy authentication. Separate with spaces. The default kubernetes network is pre-filled."
+                                        schema:
+                                          type: string
+                                          default: ""
+                                      - variable: upstream4
+                                        label: "Upstream 4"
+                                        description: "Controls which IP is written to the web server log file (requestor or x-forwarded-for header). Also allows the x-webauth-user header for proxy authentication. Separate with spaces. The default kubernetes network is pre-filled."
+                                        schema:
+                                          type: string
+                                          default: ""
```

Comment:
Uhm... nope.

You need a list. This is just a bunch of strings.
View apps in enterprise trains for examples